README.md

```diff
@@ -4,10 +4,9 @@ The Memory Analysis Cube (MAC) is an ActivePivot project that aims to provide
 the necessary tools to explore and analyze the data consumption from any other
 ActivePivot application, starting from ActivePivot 5.8 onward.
 
-The project is built as a standalone Spring Boot jar file with an embedded
-ActiveUI application available on `localhost:9092` by default.
-
-This version is compatible with ActivePivot 5.10. To use this project with other versions, pick the right branch.
+The project is built as a standalone Spring Boot jar file available on `localhost:9092` by default.
+The ActiveUI application is available in the folder ui. Enter `yarn`then `yarn start` to start ActiveUI on `localhost:9093`.
```

Comment:
Why ActiveUi cannot be deplyed like in the sandbox?

Reply:
cannot this be done with extension like in the sandbox with the mdx editor?

Reply:
I asked Antoine Tissier who told me no...